test/ecmascript.ts

```diff
@@ -264,20 +268,23 @@ describe("whole-program validation", function() {
   this.timeout(20000);
 
   // TODO(brieb): do we need these?
```

Comment:
Looks like this is the only remaining `TODO(brieb)` in this PR. Before, we ran `validateECMAScript` on this repo's source files ("main.js", "lib/shared.js", etc). How should we address these test cases, now that the source code is .ts? Should we run it on the emitted files, add different more targeted test cases as new fixtures, remove them, something else?

Reply:
~I think it would be useful to run `namedTypes.Program.assert(ast, true)` on some TypeScript ASTs, to assert that everything in the AST matches the expected types. Basically the same as `shared.validateECMAScript`, except it would only use the Babel TypeScript parser, not Esprima.~

Actually, scratch that! It looks like we already do a deep assertion for every file we process in `tests/typescript.ts` [here](https://github.com/benjamn/ast-types/blob/493d64fb0515e238a253de4bb2854247fa9a79ea/test/typescript.ts#L145), which runs over the whole https://github.com/Microsoft/TypeScript/tree/master/src/compiler codebase.

In light of that, I'm pretty sure the commented-out tests are redundant, and we can just remove them.